cluster/addons/dns/kube-dns/kube-dns.yaml.base

```diff
@@ -104,7 +115,7 @@ spec:
         kubernetes.io/os: linux
       containers:
       - name: kubedns
-        image: k8s.gcr.io/k8s-dns-kube-dns:1.14.13
+        image: k8s.gcr.io/k8s-dns-kube-dns:1.15.10
```

Comment:
perhaps have this in a separate PR in case we have to revert back again

Reply:
+1 - let's not bundle together version bump with anti-affinity thing